ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -0,0 +1,155 @@
+import logging
+
+from ocs_ci.ocs.ui.base_ui import PageNavigator
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.utility.utils import get_ocp_version, TimeoutSampler
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+
+
+logger = logging.getLogger(__name__)
+
+
+class ValidationUI(PageNavigator):
+    """
+    User Interface Validation Selenium
+
+    """
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        ocp_version = get_ocp_version()
+        self.err_list = list()
+        self.validation_loc = locators[ocp_version]["validation"]
+
+    def verify_object_service_page(self):
+        """
+        Verify Object Service Page UI
+
+        """
+        self.navigate_overview_page()
+        self.do_click(self.validation_loc["object_service_tab"])
+        platform = config.ENV_DATA.get("platform").lower()
+        if platform == constants.VSPHERE_PLATFORM:
```

Comment:
And what if platform is not vSphere but AWS?

Reply:
on AWS, these buttons are not clickable

Reply:
Do you know what is the case with BM?

Reply:
please also check Azure and RHV

Reply:
fixed